easybuild/tools/github.py

```diff
@@ -324,6 +325,22 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    if not os.path.isdir(develop):
+        raise EasyBuildError("Downloading of develop branch failed: not found in %s" % develop)
+    try:
+        os.chdir(develop)
+    except OSError:
+        _log.debug('Failed to change into directory %s: %s' % develop)
+
+    patch_name = 'patch%s' % pr
+    patch_url = URL_SEPARATOR.join([GITHUB_URL, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO, GITHUB_PR, str(pr)])
+    patch = download_file(patch_name, patch_url + '.patch', path + '/' + patch_name)
```

Comment:
this may return `None`, i.e. when downloading the patch failed, so check for that, raise an error in case it's `None`